pages/templates/pages/index.html

```diff
@@ -49,20 +49,38 @@ <h3 class="text-dark">FAQ</h3>
         </div>
         <div class="col-xs-12 col-sm-6">
             <div class="how__box how__box--big">
+                <p class="how__tittle">How does it work behind the curtain?</p>
+                <p class="how__text">The concept is simple, you generate an unique link on Hawkpost, then you share it with someone that needs to send you some secret information. When that person visits the link and submits the sensitive data, it is encrypted in their browser with your PGP public key and then forwarded to your email address. No data is stored by us.</p>
+            </div>
+        </div>
+        <div class="col-xs-12 col-sm-6">
+            <div class="how__box how__box--big">
                 <p class="how__tittle">Why should I use this?</p>
-                <p class="how__text">This is the easiest way to receive sensible information. No one needs to install anything, you share your link, the person who receive it just need to submit the information and our hawk will send it for you.</p>
+                <p class="how__text">This is the easiest way to securly receive sensible information from people who don't know how to use PGP. No one needs to install anything, you share your link, the person who receive it just need to submit the information and our hawk will send it for you.</p>
```

Comment:
@dethos: _securely_.
About the "no one needs to install anything", does that give the sense that the receiver does not need any special software?